### DISCUSSION TITLE
Overlay in-game

### DISCUSSION THREAD

**Discussion by dgenxxx** [baffler/Transparent-Twitch-Chat-Overlay#125] [General]
I'm unable to see the overlay while i'm in a game. Is there such feature in this software or if there is, how to enable it?

**Comment by q587p** [comment]
Same issue (can't see in Hunt: Showdown full-screen)

**Comment by OGNetheri** [comment]
Use Borderless Window in game and do not use Vulkan as a game's API. Twitch Chat Overlay does not dunction over full-screen at least.

**Comment by GuidiUZ** [comment]
I'm use this chat overlay in Project Zomboid in Bordeless (no full screen), and I can't see the chat. I edit the option.ini in the game too, but nothing... any help?

**Reply by OGNetheri** [reply]
Maybe it's the Isometric Engine of the game which prevents other active window's over it e.g. Twitch Chat Overlay. 

If you have option you could use some program like Display Fusion, which has option to force wanted screen always on top, in this case the Twitch Chat Overlay's? I would imagine there's free small apps for this purpose.

**Reply by GuidiUZ** [reply]
Yes, maybe it's the engine of the game... I play in window mode now, it's not the best, but I can streaming at least... thank you

**Reply by hvolschenk** [reply]
Still having this same issue.
I tried installing/using [Display Fusion](https://www.displayfusion.com/), but even that doesn't work 😅
